src/luascript.cpp

```diff
@@ -10824,6 +10871,55 @@ int LuaScriptInterface::luaPlayerGetIdleTime(lua_State* L)
 	return 1;
 }
 
+//	TempPlayer
+int LuaScriptInterface::luaTempPlayerCreate(lua_State* L)
+{
+	// TempPlayer(id or name)
+	Player* player = new Player(nullptr);
+	bool player_found = true;
```

Comment:
camelCase
also this is leaking memory.

I would suggest code:
```c++
std::unique_ptr<Player> player = std::make_unique<Player>(nullptr);
if (isNumber(L, 2)) {
	const uint32_t playerId = getNumber<uint32_t>(L, 2);
	if (!IOLoginData::loadPlayerById(player, playerId)) {
		lua_pushnil(L);
		return 1;
	}
} else if (isString(L, 2)) {
	const std::string& playerName = getString(L, 2);
	if (!IOLoginData::loadPlayerByName(player, playerName)) {
		lua_pushnil(L);
		return 1;
	}
}

if (player) {
	pushUserdata<Player>(L, player.release());
	setMetatable(L, -1, "OfflinePlayer");
} else {
	lua_pushnil(L);
}
```

Reply:
ok